gke-to-gke-vpn/cleanup.sh

```diff
@@ -93,18 +93,33 @@ if deployment_exists "${PROJECT_ID}" "cluster-deployment"; then
 fi
 
 ### Delete VPN connections
-for (( c=1; c<=4; c++ ))
-do
-  if deployment_exists "${PROJECT_ID}" "vpn$c-deployment"; then
-    gcloud deployment-manager deployments delete "vpn$c-deployment" --quiet
+timeout=100
+vpn_exists=0
+
+while [ $timeout -gt 0 ]; do
```

Comment:
Can this retry logic be made more generic into a helper function so that all deployment-manager deletes have this instead of using hardcoded "sleeps"?